src/features/profil/pages/referrals-scoreboard/components/ReferralScoreboardTable.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { XStack, YStack } from 'tamagui'
+import Text from '@/components/base/Text'
+import ProfilePicture from '@/components/ProfilePicture'
+import { ReferralScoreboardItemType } from '@/services/referral/schema'
+
+type ReferralScoreboardTableProps = {
+  data: ReferralScoreboardItemType[]
+}
+
+export default function ReferralScoreboardTable({ data }: ReferralScoreboardTableProps) {
+  return (
+    <YStack>
+      <XStack gap="$medium" p="$large" justifyContent="space-between">
+        <Text fontSize="$1" flexShrink={0}>
+          Rang
+        </Text>
+        <Text fontSize="$1" flexGrow={1}>
+          Parrain
+        </Text>
+        <Text fontSize="$1" flexShrink={0}>
+          Adhésions parrainées
+        </Text>
+      </XStack>
+
+      {data.map((entry, index) => {
+        const rowBackground = entry.is_current_user  ? '$orange1' : index % 2 ? '$white1' : '$textSurface'
```

Comment:
nous devrions revoir la config Eslint pour qu'elle corrige ce type d'"""erreur"""

```suggestion
        const rowBackground = entry.is_current_user ? '$orange1' : index % 2 ? '$white1' : '$textSurface'
```